cpp/include/raft/sparse/linalg/spmm.hpp

```diff
@@ -69,6 +69,9 @@ void spmm(raft::resources const& handle,
 
   detail::spmm(handle, trans_x, trans_y, is_row_major, alpha, descr_x, descr_y, beta, descr_z);
 
+  raft::copy(
+    z.data_handle(), z_tmp.data_handle(), z_tmp.size(), raft::resource::get_cuda_stream(handle));
+
```

Comment:
IIUC PR ( https://github.com/rapidsai/raft/pull/2124 ) ensured that this copy is no longer needed. So think we can drop this

```suggestion
```

Reply:
ha already removed! you were too quick!

https://github.com/rapidsai/raft/pull/2092/commits/c28e84752d934da772c9a40e2017be752cdfb348

Reply:
this got left behind because I just did this to get past the conflicts

```shell
git merge -X theirs branch-24.04
```

and this was the only line from that file that didn't have a conflict with this PR 🙈

Reply:
No worries. Thanks James! 🙏 

Looks good! 😄